src/test/test_win_percentage.py

```diff
@@ -0,0 +1,41 @@
+from tensorflow.python.keras.models import load_model
+import numpy as np
+
+from ..main.evaluation.win_percentage import RandomAgentEvaluator
+from ..main.environment.cross_game import CrossGame
+from ..main.models.dqn_agent import dqn_mask_loss, DQNAgent
```

Comment:
working directory : alphacross pas src/test

Reply:
Ok, ca passe chez moi. Je trouve ca plus propre de mettre ca en absolu depuis la racine, as you wish